contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -68,20 +66,18 @@
 
   private final ExcelReaderConfig readerConfig;
 
-  private XSSFSheet sheet;
+  private Sheet sheet;
 
-  private XSSFWorkbook workbook;
+  private Row currentRow;
 
-  private InputStream fsStream;
+  private Workbook workbook;
 
-  private FormulaEvaluator evaluator;
+  private InputStream fsStream;
 
   private ArrayList<String> excelFieldNames;
```

Comment:
Use List instead of ArrayList here and in other fields below.

Reply:
Fixed